src/js/stanford_basic.behavior.js

```diff
@@ -25,6 +25,11 @@ export default {
       if (!$sn) {
         $('.su-skipnav--secondary', context).remove();
       }
+
+      // Move the second search block to the navigation. Hide it and then only show for mobile sites.
+      var $ms = $('nav + .su-site-search');
+      $($ms).prependTo('.su-masthead .su-multi-menu > ul').wrap('<li></li>');
```

Comment:
```suggestion
      var $ms = $('.su-masthead .su-site-search', context);
      if ($ms.length == 1) {
        $($ms).prependTo('.su-masthead .su-multi-menu > ul', context).wrap('<li></li>');
      }
```

A couple of changes here. When in the `attach` behavior in Drupal you should be using the `context` option within you jquery query statements. Otherwise, this function will trigger repeatedly each time an ajax call happens. The context bounds the code to run only on the content that is changing. We also cannot assume that all sites going forward will have the search box in their masthead. We should check for it.